Automattic/woocommerce-payments
# Add screenshots for wp.org listing

Fixes #662

#### Changes proposed in this Pull Request

Add screenshots for wp.org listing. Screenshots are based on WooCommerce plugin listing designs.

#### Testing instructions

I'm not sure how to test screenshots without deploying to wp.org. A smoke test is just to view images and confirm they're in the right directory.

-------------------
